.circleci/config.yml

```
@@ -48,9 +48,47 @@ jobs:
             sudo mv /tmp/kubebuilder_2.0.0-alpha.4_${os}_${arch} /usr/local/kubebuilder
             export PATH=$PATH:/usr/local/kubebuilder/bin
       - run: make test
+
+  release:
+    docker:
+      - image: circleci/golang:1.12
+        environment:
+          - GO111MODULE=on
+    working_directory: /go/src/github.com/ory/oathkeeper-k8s-controller
+    steps:
+      - checkout
+      - run:
+          command: |
+            echo 'export DOCKER_FULL_TAG=$(echo $CIRCLE_TAG | tr '+' '_')' >> $BASH_ENV
+            echo 'export DOCKER_SHORT_TAG=$(echo $CIRCLE_TAG | cut -d '+' -f1)' >> $BASH_ENV
+            ./.circleci/release_name.bash
+            source $BASH_ENV
+      - setup_remote_docker
+      - run: docker login --username "$DOCKER_USERNAME" --password "$DOCKER_PASSWORD"
+      - run: curl -sL https://git.io/goreleaser | bash
```

Add line break for readability

```suggestion
      - run: curl -sL https://git.io/goreleaser | bash

`````